server/shadow/shadow_server.c

```diff
@@ -45,10 +45,15 @@
 
 #define TAG SERVER_TAG("shadow")
 
+static const char bind_address[] = "bind-address,";
+
 static const COMMAND_LINE_ARGUMENT_A shadow_args[] = {
 	{ "port", COMMAND_LINE_VALUE_REQUIRED, "<number>", NULL, NULL, -1, NULL, "Server port" },
 	{ "ipc-socket", COMMAND_LINE_VALUE_REQUIRED, "<ipc-socket>", NULL, NULL, -1, NULL,
 	  "Server IPC socket" },
+	{ "bind-address", COMMAND_LINE_VALUE_REQUIRED, "<bind-address>", NULL, NULL, -1, NULL,
+	  "An address to bind to. Use '[<ipv6>]' for IPv6 addresses, e.g. '[::]' for "
+	  "localhost" },
```

Comment:
This is incorrect, `::` is all addresses and `::1` is localhost

Reply:
good catch